packages/graphql-elasticsearch-transformer/src/SearchableModelTransformer.ts

```diff
@@ -27,12 +27,12 @@ import path = require('path');
 const STACK_NAME = 'SearchableStack';
 const nonKeywordTypes = ["Int", "Float", "Boolean", "AWSTimestamp", "AWSDate", "AWSDateTime"];
```

Comment:
Question: Do we have all the scalar types here, what we need?https://docs.aws.amazon.com/appsync/latest/devguide/scalars.html

Reply:
This is list is a subset of the scalar types which are not serialized as strings and types which may be serialized as strings in ddb though they are set to a different type in ES. (For example AWSDate is serialized as a string though it is picked up as a date type in ES)